chain/chain/src/test_utils/kv_runtime.rs

```diff
@@ -440,7 +441,42 @@ impl EpochManagerAdapter for KeyValueRuntime {
     }
 
     fn get_epoch_info(&self, _epoch_id: &EpochId) -> Result<Arc<EpochInfo>, Error> {
```

Comment:
Please add a comment explaining which parts of the `EpochInfo` are real, and which ones are fake, and that `epoch_id` isn't used in the tests.

`epoch_height=10` I copied from some other test function building `EpochInfo`, but I don't remember which one :thinking: